packages/console/src/ds-components/DataTransferBox/SourceGroupItem/index.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import CaretExpanded from '@/assets/icons/caret-expanded.svg';
+import CaretFolded from '@/assets/icons/caret-folded.svg';
+import Checkbox from '@/ds-components/Checkbox';
+import IconButton from '@/ds-components/IconButton';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import SourceDataItem from '../SourceDataItem';
+import { type DataEntry, type DataGroup, type SelectedDataEntry } from '../type';
+
+import * as styles from './index.module.scss';
+
+type Props<TEntry extends DataEntry> = {
+  dataGroup: DataGroup<TEntry>;
+  selectedGroupDataList: Array<SelectedDataEntry<TEntry>>;
+  onSelectDataGroup: (group: DataGroup<TEntry>) => void;
+  onSelectData: (data: TEntry) => void;
```

Comment:
Better add some comments here to explain the difference between data and data group, as it confuses me a little bit

Reply:
sure

Reply:
updated